spec/support/generators.rb

```diff
@@ -0,0 +1,22 @@
+module Pageflow
+  module Specs
+    module Generators
+      module Macros
```

Comment:
The automatic loading of files is also something frowned upon these days ... makes everything slow. That's why I required it just in the tests.